tsl/test/expected/continuous_aggs_bgw_drop_chunks.out

```
@@ -0,0 +1,123 @@
+ set_chunk_time_interval 
+-------------------------
+ 
+(1 row)
+
+\set ON_ERROR_STOP 0
+INSERT INTO drop_chunks_table SELECT i, i FROM generate_series(1, 39) AS i;
+\set ON_ERROR_STOP 1
+REFRESH MATERIALIZED VIEW drop_chunks_view1;
+INFO:  materializing continuous aggregate public.drop_chunks_view1: nothing to invalidate, new range up to 45
+--TEST1  specify drop chunks policy on mat. hypertable by
+-- directly does not work
+\set ON_ERROR_STOP 0
+SELECT add_drop_chunks_policy( '_timescaledb_internal._materialized_hypertable_2', older_than=> -50, cascade_to_materializations=>false ) as drop_chunks_job_id1 \gset
+WARNING:  Timescale License expired
+ERROR:  cannot add drop chunks policy to materialized hypertables "_materialized_hypertable_2" 
+\set ON_ERROR_STOP 1
+--TEST2  specify drop chunks policy on cont. aggregate
+-- integer_now func on raw hypertable is used by the drop
+-- chunks policy
+SELECT hypertable_id, table_name, integer_now_func
+FROM _timescaledb_catalog.dimension d,  _timescaledb_catalog.hypertable ht
+WHERE ht.id = d.hypertable_id;
+ hypertable_id |         table_name         | integer_now_func  
+---------------+----------------------------+-------------------
+             1 | drop_chunks_table          | integer_now_test2
+             2 | _materialized_hypertable_2 | 
+(2 rows)
+
+SELECT chunk_table, ranges FROM chunk_relation_size('_timescaledb_internal._materialized_hypertable_2')
+ORDER BY ranges;
+               chunk_table               |   ranges    
+-----------------------------------------+-------------
+ _timescaledb_internal._hyper_2_40_chunk | {"[0,10)"}
+ _timescaledb_internal._hyper_2_41_chunk | {"[10,20)"}
+ _timescaledb_internal._hyper_2_42_chunk | {"[20,30)"}
+ _timescaledb_internal._hyper_2_43_chunk | {"[30,40)"}
+(4 rows)
+
+SELECT add_drop_chunks_policy( 'drop_chunks_view1', older_than=> 10, cascade_to_materializations=>false ) as drop_chunks_job_id1 \gset
```

What would `cascade_to_materializations=true` mean if this is executed on a view? Does it just silently do nothing or does it error out?

---

it silently does nothing now.